lib/aws/awsconfig.js

```diff
@@ -20,6 +20,73 @@ function AWSConfig() {
     return configService;
   }
 
+  me.enableRule = function(input) {
+    var self = arguments.callee;
+    var configService = me.findService(input);
+    if(input.owner == "CUSTOM_LAMBDA"){
+    var params = {
```

Comment:
I'm not sure what you mean. What I'm trying to point out is that the only difference is to set 'SourceDetails' inside ConfigRule.Source.